libbeat/esleg/eslegclient/bulkapi_mock_test.go

```diff
@@ -36,7 +36,10 @@ import (
 )
 
 func TestOneHostSuccessResp_Bulk(t *testing.T) {
-	logp.TestingSetup(logp.WithSelectors("elasticsearch"))
+	setupErr := logp.TestingSetup(logp.WithSelectors("elasticsearch"))
+	if setupErr != nil {
+		t.Fatal(setupErr)
+	}
```

Comment:
You could achieve the same effect with:
```suggestion
	require.NoError(setupErr)
```